pt_miniscreen/app.py

```diff
@@ -113,6 +113,10 @@ def handle_button_press(button: MiniscreenButton):
                 f"Handling button {button} for tile group {self.current_tile_group}"
             )
 
+            if not getattr(self, "state_manager"):
```

Comment:
This throws an AttributeException if you don't provide a default
```suggestion
            if not getattr(self, "state_manager", None):
```